legged_unitree_hw:
  loop_frequency: 400 # 500
  cycle_time_error_threshold: 0.006 # 0.002
  thread_priority: 95
  power_limit: 4
  contact_threshold: 40 #40
